### PR TITLE
presto-blackhole: add page processing delay

### PR DESCRIPTION
**Pull request by arhimondr**
"page_processing_delay" blackhole table property added.
With this property delay between pages processing can be specified.

Testing: unit test

### CODE REVIEW

**Comment by kokosing** [comment]
looks good

**Comment by pnowojski** [comment]
"Might" be usefull? If it's not, then why do you implement it? 

I don't understand why do you need it? What's wrong with "select avg(orderkey) from blackhole.default.very_large_line_item"? We wouldn't need to maintain those 261 lines of code.

**Comment by arhimondr** [comment]
`select avg(orderkey) from blackhole.default.very_large_line_item` will cause high CPU/Memory/Network consumption.

File: presto-docs/src/main/sphinx/connector/blackhole.rst [hunk 1]
```diff
@@ -79,3 +79,29 @@ table property (default value is equal to 16)::
       rows_per_page = 2000,
       field_length = 100
     );
+
```
Comment (pnowojski) [inline presto-docs/src/main/sphinx/connector/blackhole.rst:85]: I would change first paragraph of this doc. Replace:

The Black Hole connector works like the `/dev/null` device on Unix-like
operating systems for data writing and like `/dev/null` or `/dev/zero`
for data reading.

with:

The Black Hole connector is a testbed for performing various kinds of tests. Primarily it is designed for high performance testing of other components. Thus it works like the `/dev/null` device on Unix-like operating systems for data writing and like `/dev/null`or `/dev/zero` for data reading. However it has also some other features that allows testing Presto in a more controlled manner.

**Pull request merged**